pkgs/applications/audio/faust/faust2.nix

```diff
@@ -16,11 +16,13 @@ with stdenv.lib.strings;
 
 let
 
-  version = "2.0.a51";
+  version = "2-1-0";
```

Comment:
While upstream uses this tag, I don't think this is a valid version for in Nixpkgs and so recommend to use instead "2.1.0".